src/screens/SettingsModal.js

```diff
@@ -203,9 +208,9 @@ export default function SettingsModal() {
                   name={key}
                   options={{
                     cardStyleInterpolator,
-                    title,
+                    title: title(),
                   }}
-                  title={title}
+                  title={title()}
```

Comment:
nit, feels a bit weird seeing `title()` - could we rename this param to `getTitle` instead